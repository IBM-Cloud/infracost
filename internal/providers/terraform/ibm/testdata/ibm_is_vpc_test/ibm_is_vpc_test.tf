
terraform {
  required_providers {
    ibm = {
      source = "IBM-Cloud/ibm"
    }
  }
}

provider "ibm" {
  region = "us-south"
}

resource "ibm_is_vpc" "testVpc" {
  name = "test-vpc"
}

resource "ibm_is_vpc" "test_vpc_without_usage" {
  name = "test-vpc-without-usage"
}

