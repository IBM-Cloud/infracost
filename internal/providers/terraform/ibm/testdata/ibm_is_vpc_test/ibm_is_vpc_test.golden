
 Name                Monthly Qty  Unit      Monthly Cost 
                                                         
 ibm_is_vpc.testVpc                                      
 └─ VPC us-south               1  Instance         $0.00 
                                                         
 OVERALL TOTAL                                     $0.00 
──────────────────────────────────
1 cloud resource was detected:
∙ 1 was estimated, it includes usage-based costs, see https://infracost.io/usage-file